### ISSUE TITLE
Only uses the default AWS profile

### ISSUE THREAD

**Issue by hassankhan** [arminhammer/bellerophon#2]
Is there any way to allow for switching between profiles? Thanks for writing this, by the way :)

**Comment by arminhammer** [comment]
Hey hassankhan, this sounds like a good feature, thanks for submitting! Maybe it could be part of a new settings tab? I'll add it as a TODO item for the next version, 0.3.0.

**Comment by hassankhan** [comment]
Yeah, or possibly exposed as an option from the Menu bar (File, Edit) etc? Would love a "quick-switcher" type thing.
